src/client/java/build.sh

```diff
@@ -51,10 +53,9 @@ setup_graph_source()
 #-----------------------------------------------
 compile_java_client()
 {
-	mvn clean package -DJAVA_FBTHRIFT_JAR=$java_fbthrift_jar -f ${pom_file_path}
```

Comment:
If mvn repo doesn't exist `fbthrift` dependency, `Java-client` can't be compiled successfully. So should add `mvn install::install-file ...` command to install `fbthrift` into local mvn repo before this command.

Reply:
I  have add this in read.me `mvn install ` is ok.